lib/dfe/reference_data/equality_and_diversity/disabilities.rb

```diff
@@ -0,0 +1,112 @@
+          { name: 'Blindness or a visual impairment not corrected by glasses',
+            hesa_code: {
+              '2019-2020' => '58',
+              '2020-2021' => '58',
+              '2021-2022' => '58',
+              '2022-2023' => '58'
+            } },
+          '82e2ad24-5568-4e0b-a362-41666a668fb9' =>
+          { name: 'Condition affecting motor, cognitive, social and emotional skills, speech or language since childhood',
+            hesa_code: {
+              '2022-2023' => '59'
+            } },
+          'c6302624-b33a-46fb-8dcb-058f21cee0dc' =>
+          { name: 'Deafness or a serious hearing impairment',
+            hesa_code: {
+              '2019-2020' => '57',
+              '2020-2021' => '57',
+              '2021-2022' => '57',
+              '2022-2023' => '57'
+            } },
+          'f9624005-d7aa-45b3-bfce-ef2e2779f631' =>
+          { name: 'Dyslexia, dyspraxia or attention deficit hyperactivity disorder (ADHD) or another learning difference',
+            hesa_code: {
+              '2019-2020' => '51',
+              '2020-2021' => '51',
+              '2021-2022' => '51',
+              '2022-2023' => '51'
+            } },
+          '9955ea7d-e147-4b12-8913-d0c4ec925409' =>
+          { name: 'Long-term illness',
+            hint: 'For example, cancer, HIV, diabetes, chronic heart disease or epilepsy',
+            hesa_code: {
+              '2019-2020' => '54',
+              '2020-2021' => '54',
+              '2021-2022' => '54',
+              '2022-2023' => '54'
+            } },
+          '2dc8d721-a543-4ec4-aec4-9cc22fe1ebeb' =>
+          { name: 'Mental health condition',
+            hint: 'For example, depression, schizophrenia or anxiety disorder',
+            hesa_code: {
+              '2019-2020' => '55',
+              '2020-2021' => '55',
+              '2021-2022' => '55',
+              '2022-2023' => '55'
+            } },
+          '79785b6c-d4f3-4b01-a13a-35ac8174b789' =>
+          { name: 'Physical disability or mobility issue',
+            hint: 'For example, impaired use of arms or legs, use of a wheelchair or crutches',
+            hesa_code: {
+              '2019-2020' => '56',
+              '2020-2021' => '56',
+              '2021-2022' => '56',
+              '2022-2023' => '56'
+            } },
+          '3451285e-972b-464c-9726-84cae27b82ea' =>
+          { name: 'Another disability, health condition or impairment affecting daily life',
+            generic: true,
+            hesa_code: {
+              '2019-2020' => '96',
+              '2020-2021' => '96',
+              '2021-2022' => '96',
+              '2022-2023' => '96'
+            } },
+          'd3f0f6de-b9be-4299-ade0-b40eef5d9ef2' =>
+          { name: 'Prefer not to say',
+            opt_out: true,
+            hesa_code: {
+              '2019-2020' => '98',
+              '2020-2021' => '98',
+              '2021-2022' => '98',
+              '2022-2023' => '98'
+            } } },
```

Comment:
I wonder if this should be grouped separately. Certainly in our UI's we'll handle them separately.